src/machine.rs

```diff
@@ -135,6 +135,17 @@ impl MayLeak for MiriMemoryKind {
     }
 }
 
+impl MiriMemoryKind {
+    /// Whether we have a useful allocation span for an allocation of this kind.
+    fn should_save_allocation_span(self) -> bool {
+        use self::MiriMemoryKind::*;
+        match self {
+            Rust | Miri | C | Mmap => true,
+            Machine | Global | ExternStatic | Tls | WinHeap | Runtime => false,
```

Comment:
Why is `WinHeap` treated different from the other heap allocations?

Reply:
I probably pulled it into the false list then never really noticed the mistake... do we have any tests that would show a use-after-free span for WinHeap? I don't think we actually do.

Reply:
I don't think we have such a test, no.